shell/common/engine.cc

```diff
@@ -133,6 +134,7 @@ std::unique_ptr<Engine> Engine::Spawn(
       settings_.isolate_shutdown_callback,   // isolate shutdown callback
       settings_.persistent_isolate_data      // persistent isolate data
   );
+  result->initial_route_ = std::move(initial_route);
```

Comment:
I don't think you want an std::move here.  This has to initiate a copy anyways, I think it is just needlessly invalidating the reference.

Reply:
Done